docker-compose.yml

```diff
@@ -1,114 +1,96 @@
-version: '3.7'
-
-volumes:
-  pg_data:
-
+---
 services:
-  solr:
-    image: solr:8-slim
+  app:
+    depends_on:
+      - db
+      - solr
+      - redis
+    image: umich-arclight
+    build:
+      dockerfile: Dockerfile
+      context: .
     ports:
-      - "8983:8983"
+      - "3000:3000" # Rails
+      - "1234:1234" # RubyMine
+      - "26162:26162" # RubyMine
     environment:
-      - SOLR_JAVA_MEM=-Xms1024m -Xmx1024m
-      - SOLR_HEAP=1024m
+      - RAILS_ENV=development
+      - REDIS_URL=redis://redis:6379
+      - DATABASE_URL=postgresql://postgres:postgres@db/umich-arclight-development
+      - SOLR_URL=http://solr:8983/solr/umich-arclight-development
     volumes:
-#      - ./solr/arclight:/var/solr/data/arclight
-      - ./solr/arclight/conf:/solr_config:ro
+      - .:/opt/app
+      - ./sample-ead:/data:ro
```

Comment:
If you're mirroring production, `/data` cannot be `ro`; the PDF workflow writes to `$FINDING_AID_DATA/pdf`.

Although that's missing too?

Reply:
The default FINDING_AID_DATA is /data but you need the environment variable, I understand.

Reply:
The default is fine, but I need it rw

Reply:
Declaration
```
services:
  app:
    volumes:
      - data:/var/opt/app/data
```
Initialization
```
volumes:
  data:
    driver_opts:
      type: none
      device: ${PWD}/sample-ead
      o: bind
```

Reply:
Changed my mind and decide NOT to hack the "data" volume to do automagically sync with the local `./sample-ead` directory. 

```
volumes:
    data:
```